frontend/src/common/queries/filter.ts

```diff
@@ -277,12 +278,16 @@ function aggregateCollectionDatasetRows(
           ...collectionDatasetRow.development_stage_ancestors,
         ],
         disease: [...accum.disease, ...collectionDatasetRow.disease],
+        organism: [...accum.organism, ...collectionDatasetRow.organism],
```

Comment:
Corrected `ESLint: Expected object keys to be in ascending order. 'organism' should be before 'self_reported_ethnicity'.(sort-keys)` linting error.